synapse/models/risk.py

```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
 
                     ('name', ('str', {'lower': True, 'onespace': True}), {
                         'ex': "apt1 (mandiant)",
-                        'doc': 'The name of the threat cluster.'}),
+                        'doc': 'A brief descriptive name for the threat cluster.'}),
 
                     ('type', ('risk:threat:type:taxonomy', {}), {
-                        'doc': 'The type of threat cluster.'}),
+                        'doc': 'A user-specified type for the threat.'}),
 
                     ('desc', ('str', {}), {
                         'doc': 'A description of the threat cluster.'}),
 
                     ('tag', ('syn:tag', {}), {
-                        'doc': 'The tag used to annotate nodes that are members of the cluster.'}),
+                        'doc': 'The tag used to annotate nodes that are associated with the threat cluster.'}),
 
                     ('reporter', ('ou:org', {}), {
-                        'doc': 'The organization who published the threat cluster.'}),
+                        'doc': 'The organization reporting on the threat cluster.'}),
 
                     ('reporter:name', ('ou:name', {}), {
-                        'doc': 'The name of the organization who published the threat cluster.'}),
+                        'doc': 'The name of the organization reporting on the threat cluster.'}),
 
                     ('org', ('ou:org', {}), {
-                        'doc': 'The organization that the threat cluster is attributed to.'}),
+                        'doc': 'An authoritative organization for the threat cluster.'}),
 
                     ('org:loc', ('loc', {}), {
-                        'doc': 'The assessed location of the organization that the threat cluster is attributed to.'}),
+                        'doc': 'The reporter\'s assessed location of the threat cluster.'}),
```

Comment:
Gets a bit wordy, but may want to consider "The reporting orgnizations's" here and below?